modules/installation-custom-gcp-vpc.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-vpc.adoc
+
+:_content-type: CONCEPT
+[id="installation-custom-gcp-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into existing subnets in an existing Virtual Private Cloud (VPC) in Google Cloud Platform (GCP). By deploying {product-title} into an existing GCP VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option.
+
+Because the installation program cannot know what other components are also in your existing subnets, it cannot choose subnet CIDRs and so forth on your behalf. You must configure networking for the subnets that you install your cluster to yourself.
```

Comment:
> Because the installation program cannot know what other components are also in your existing subnets, it cannot choose subnet CIDRs and so forth on your behalf.

What does this mean? The installer does not choose subnet CIDRs because the installer is not creating the subnets. A subnet has exactly one CIDR.

Reply:
Addressed. Removed the ambiguous portion

Reply:
Addressed. Removed the ambiguous portion